chain/store.go

```diff
@@ -52,6 +53,19 @@ const (
 	PostgreSQL StorageType = "postgres"
 )
 
+func MetricsStorageType(st StorageType) int {
+	// NOTE: Please only append to this list.
+	switch st {
+	case BoltDB:
+		return 1 // We know this to be Bolt
+	case PostgreSQL:
+		return 2 //nolint:gomnd // We know this to be Postgres
```

Comment:
How do we know it? Please cross-link with the other concerned files in the comments.

Reply:
That was a comment for the linter. I've refactored this to use const/iota per @CluEleSsUK's comment above.